### PR TITLE
minor: PhpdocAlignFixerTest - convert CUSTOM tags test to not rely on non-custom tag from TAGS_WITH_NAME

### PR DESCRIPTION
**Pull request by keradus**
minor appendix i came up after checking #7120

### CODE REVIEW

**Comment by Wirone** [comment]
I don't get the point of this PR, maybe you wanted to be like Elon and introduce even more X'es 😆.

**Comment by keradus** [comment]
the test name claims we test ` 'custom tags'`. but actually, we do not test anything custom here, only the tags that are well defined in `TAGS_WITH_NAME` structure. so it was not testing anything custom

File: tests/Fixer/Phpdoc/PhpdocAlignFixerTest.php [hunk 1]
```diff
@@ -1284,13 +1284,13 @@ class Foo {}
         ',
         ];
 
-        yield 'custom tags' => [
-            ['tags' => ['param', 'phpstan-param']],
+        yield 'CUSTOM tags' => [
+            ['tags' => ['param', 'xxxxxxxxxxxxx']],
```
Comment (kubawerlos) [inline tests/Fixer/Phpdoc/PhpdocAlignFixerTest.php:1288]: Can we not lose here tag with dash?

File: tests/Fixer/Phpdoc/PhpdocAlignFixerTest.php [hunk 2]
```diff
@@ -1300,7 +1300,7 @@ class Foo {}
     /**
      * @param  EngineInterface $templating
      * @param string      $format
-     * @phpstan-param  int  $code       An HTTP response status code
+     * @xxxxxxxxxxxxx  int  $code       An HTTP response status code
```
Comment (keradus) [inline tests/Fixer/Phpdoc/PhpdocAlignFixerTest.php:1303]:
```suggestion
     * @x-xxxxxxxxxxx  int  $code       An HTTP response status code
```

File: tests/Fixer/Phpdoc/PhpdocAlignFixerTest.php [hunk 3]
```diff
@@ -1284,13 +1284,13 @@ class Foo {}
...
-        yield 'custom tags' => [
-            ['tags' => ['param', 'phpstan-param']],
+        yield 'CUSTOM tags' => [
+            ['tags' => ['param', 'xxxxxxxxxxxxx']],
             '<?php
     /**
      * @param         EngineInterface $templating
      * @param         string          $format
-     * @phpstan-param int             $code       An HTTP response status code
+     * @xxxxxxxxxxxxx int             $code       An HTTP response status code
```
Comment (keradus) [inline tests/Fixer/Phpdoc/PhpdocAlignFixerTest.php:1293]:
```suggestion
     * @x-xxxxxxxxxxx int             $code       An HTTP response status code
```

File: tests/Fixer/Phpdoc/PhpdocAlignFixerTest.php (hunk 1 above)
Comment (keradus) [reply tests/Fixer/Phpdoc/PhpdocAlignFixerTest.php:1288]: sure!
```suggestion
            ['tags' => ['param', 'xxx-xxxxxxxxx']],
```

File: tests/Fixer/Phpdoc/PhpdocAlignFixerTest.php [hunk 4]
```diff
@@ -1284,13 +1284,13 @@ class Foo {}
         ',
         ];
 
-        yield 'custom tags' => [
-            ['tags' => ['param', 'phpstan-param']],
+        yield 'CUSTOM tags' => [
```
Comment (kubawerlos) [inline tests/Fixer/Phpdoc/PhpdocAlignFixerTest.php:1287]: Non-blocking: what is the reasoning for making "CUSTOM" all caps?

**Review by kubawerlos — APPROVED** [review]

**Pull request merged**